### ISSUE TITLE
SNMP Plugin: Handling of String Values

### ISSUE THREAD

**Issue by DonRichie** [influxdata/telegraf#1361]
Hello,

at first:
Client-Linux: SLES12
Telegraf Version: 0.13.1

I am trying to use Telegraf to gather data from an EMC Data Domain via SNMP 

This is my configuration:

```
[[inputs.snmp.host]]
    address = "fqdn:161"
    community = "public" # default public
    version = 2 # default 2
    timeout = 1000.0 # default 2.0
    retries = 2 # default 2
    collect = ["fileSystemResourceName", "fileSystemSpaceSize", "fileSystemSpaceUsed", "fileSystemSpaceAvail", "fileSystemPercentUsed"]

[[inputs.snmp.bulk]]
     name = "fileSystemResourceName"
     oid = ".1.3.6.1.4.1.19746.1.3.2.1.1.3"
[[inputs.snmp.bulk]]
     name = "fileSystemSpaceSize"
     oid = ".1.3.6.1.4.1.19746.1.3.2.1.1.4"
[[inputs.snmp.bulk]]
     name = "fileSystemSpaceUsed"
     oid = ".1.3.6.1.4.1.19746.1.3.2.1.1.5"
[[inputs.snmp.bulk]]
     name = "fileSystemSpaceAvail"
     oid = ".1.3.6.1.4.1.19746.1.3.2.1.1.6"
[[inputs.snmp.bulk]]
     name = "fileSystemPercentUsed"
     oid = ".1.3.6.1.4.1.19746.1.3.2.1.1.7"
```

Unfortunately only the "fileSystemPercentUsed" is showing up in my Influx-Database.

My guess why it comes to this behavior is that the percent value is marked as Integer while the others are not:
```
[...]
DATA-DOMAIN-MIB::fileSystemSpaceAvail.1 = STRING: 123.2
DATA-DOMAIN-MIB::fileSystemSpaceAvail.2 = STRING: 123.5
DATA-DOMAIN-MIB::fileSystemSpaceAvail.4 = STRING: 123.4
[...]
DATA-DOMAIN-MIB::fileSystemPercentUsed.1 = INTEGER: 10
DATA-DOMAIN-MIB::fileSystemPercentUsed.2 = INTEGER: 20
[...]
```

What can I do about this problem?

Another question I have is the following:
There are special values containing names:
```
DATA-DOMAIN-MIB::mtreeRetentionLockMtreeName.0 = STRING: /data/a
DATA-DOMAIN-MIB::mtreeRetentionLockMtreeName.1 = STRING: /data/b
```
The "0" and "1" are the instance number which are common to the other values.
How can I use these values to for example tag my data so I can display it later in Grafana?

Thanks you for your effort and regards.

**Comment by sparrc** [comment]
this is now available in 1.0 RC 1 with the new SNMP plugin, please see here to download the latest version: https://github.com/influxdata/telegraf#linux-deb-and-rpm-packages, and [the changelog](https://github.com/influxdata/telegraf/blob/master/CHANGELOG.md) for details on the new SNMP plugin.

**Comment by DonRichie** [comment]
Oh oh oh!!! Awesome.
I will use this. Thank you very much
